manifests/cluster/deployment.yaml

```diff
@@ -4126,6 +4126,28 @@ spec:
                         - authenticationURL
                         - container
                         type: object
+                      oss:
+                        properties:
+                          accessKeyID:
+                            type: string
+                          accessSecretRef:
+                            type: string
+                          bucket:
+                            type: string
+                          endpoint:
+                            type: string
+                          pathPrefix:
+                            type: string
+                          serverSideEncryption:
```

Comment:
Why is it defined inside chartmuseum CRD while it seems that chartmuseum doesn't use it ?

Reply:
yes it is not used at all, just refer what ChartMuseumChartStorageDriverAmazonSpec did, maybe for future use.

Reply:
Sorry, I don't understand, why do we define it if we don't use it ? It's only defined for amazon, not for gcs...